tests/unit/test_table.py

```diff
@@ -1340,16 +1400,21 @@ def _list_backups_helper(self, cluster_id=None, filter_=None, **kwargs):
             cluster_id = "-"
         parent = "{}/clusters/{}".format(self.INSTANCE_NAME, cluster_id)
 
-        expected_metadata = [
-            ("x-goog-request-params", "parent={}".format(parent)),
-        ]
+        order_by = None
+        page_size = 0
+        if "order_by" in kwargs:
+            order_by = kwargs["order_by"]
+
+        if "page_size" in kwargs:
+            page_size = kwargs["page_size"]
+
         api.assert_called_once_with(
-            bigtable_table_admin_pb2.ListBackupsRequest(
-                parent=parent, filter=backups_filter, **kwargs
-            ),
-            retry=mock.ANY,
```

Comment:
@crwilcox one last thing to point out - the last few params were now removed from the assertion because we are no longer using the `inner_api_calls` to get the result, let me know if oyu think this should still be here.